mscore/CMakeLists.txt

```diff
@@ -682,9 +682,9 @@ endif (APPLE)
 
 # Add the appropriate splash screen for the build
 if (MSCORE_UNSTABLE)
-    cp -f /assets/splash-dev.png  mscore/data/splash.png # Dev splash on development builds
+    install(FILES ../assets/splash-dev.png RENAME splash.png DESTINATION ${PROJECT_BINARY_DIR}/mscore/data) # Dev splash on development builds
```

Comment:
It's the [other command](https://github.com/musescore/MuseScore/pull/3269/commits/4000c16bbf98d7defb0efa28276d0e11257f4791#r191898009), sorry. My mistake.

Reply:
Okay thanks, committing now.

Reply:
What you've written here is valid CMake, so it will compile without errors and probably pass the Travis CI tests, but the splash image will be wrong because it was copied *after* the executable had finished compiling.

Reply:
Oh, so I need to put it earlier in the file.

Reply:
So I need to put it earlier and add the relative directory dots?

Reply:
No, here is OK, but you need to [use `configure_file` rather than `install`](https://github.com/musescore/MuseScore/pull/3269/commits/4000c16bbf98d7defb0efa28276d0e11257f4791#r191898009).

Reply:
Is it relative for both or just the first?